pkg/log/log.go

```diff
@@ -0,0 +1,142 @@
+package log
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/go-logr/logr"
+	"go.uber.org/zap/zapcore"
+	"k8s.io/klog/v2"
+	ctrl "sigs.k8s.io/controller-runtime"
+	ctrllog "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+var (
+	// Log is the base logger
+	Log logr.Logger = ctrllog.NullLogger{}
+)
```

Comment:
I don't think we require an upgrade here, right?

Reply:
we will when upgrading logr to 1.X